blocks/medium-promo-block/features/medium-promo/default.jsx

```diff
@@ -20,83 +66,150 @@ const MediumPromoItem = ({ customFields, arcSite }) => {
 			// does not need embed_html because no video section
 			// does not need website section nor label because no overline
 			filter: `{
-      _id
-      credits {
-        by {
-          _id
-          name
-          url
-          type
-          additional_properties {
-            original {
-              byline
-            }
-          }
-        }
-      }
-      description {
-        basic
-      }
-      display_date
-      type
-      headlines {
-        basic
-      }
-      promo_items {
-        type
-        url
-        lead_art {
-          type
-          promo_items {
-            basic {
-              type
-              url
-              resized_params {
-                400x300
-                400x267
-                400x225
-                274x206
-                274x183
-                274x154
-              }
-            }
-          }
-        }
-        basic {
-          type
-          url
-          resized_params {
-            400x300
-            400x267
-            400x225
-            274x206
-            274x183
-            274x154
-          }
-        }
-      }
-      websites {
-        ${arcSite} {
-          website_url
-        }
-      }
-    }`,
+			_id
+			credits {
+				by {
+					_id
+					name
+					url
+					type
+					additional_properties {
+						original {
+							byline
+						}
+					}
+				}
+			}
+			description {
+				basic
+			}
+			display_date
+			type
+			headlines {
+				basic
+			}
+			promo_items {
+				type
+				url
+				lead_art {
+					_id
+					auth {
+						${RESIZER_APP_VERSION}
+					}
+					type
+					promo_items {
+						basic {
```

Comment:
Is `promo_items` ever used? If so, we need to add `auth` and `_id` here otherwise it will break

Reply:
@matthewroach : This one is for `[content].lead_art.promo_items`. So far I haven't had a use case where `auth` or `_id` are needed here, but let me know if I'm overlooking. I've the `_id` and `auth` included for:

1. `[content].lead_art`
2.`[content].promo_items.basic`

Those seem to be the two cases we're covering for a story's Featured Media.

Reply:
Correct that's the location - Does the block ever need that content? If it's not using it we can remove these items from the `filter` but are we 100% sure the block isn't supposed to be using it? Are we introducing a regression? or are we missing a test case?

Reply:
This block doesn't use these fields. I will remove and test to ensure I'm not introducing a regression, also checking v1 of the block.

Reply:
It was needed - I've updated the filter to match the different items needed